server/embed/config.go

```diff
@@ -762,6 +770,9 @@ func (cfg *Config) AddFlags(fs *flag.FlagSet) {
 	// unsafe
 	fs.BoolVar(&cfg.UnsafeNoFsync, "unsafe-no-fsync", false, "Disables fsync, unsafe, will cause data loss.")
 	fs.BoolVar(&cfg.ForceNewCluster, "force-new-cluster", false, "Force to create a new one member cluster.")
+
+	// featuregate
+	cfg.ServerFeatureGate.(featuregate.MutableFeatureGate).AddFlag(fs, ServerFeatureGateFlagName)
```

Comment:
We haven't finalized how to proceed with the so-called "cluster-level" feature.
```suggestion
	cfg.ServerFeatureGate.(featuregate.MutableFeatureGate).AddFlag(fs, "feature-gates")
```

Reply:
`ServerFeatureGateFlagName` is referenced in another place as well. We would want different flags for `server-feature-gates` and `cluster-feature-gates`.

Reply:
We need to clarify semantics of feature gates to ensure that flag names properly reflect that. Please see my comment. https://github.com/kubernetes/enhancements/pull/4662#discussion_r1662333970

 Still until we cut v3.6 we can always change the name. Let's base our current naming on fact that we are only implementing server level flags and that KEP for cluster level features is not yet landed.

Reply:
ok. Keeping the flag name `feature-gates` for now.